Packs/CiscoEmailSecurity/Integrations/CiscoEmailSecurity/CiscoEmailSecurity.yml

```diff
@@ -0,0 +1,1224 @@
+      description: Spam positive to filter with.
+      isArray: false
+      name: spam_positive
+      predefined:
+      - 'True'
+      - 'False'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: In Spam Quarantine, to filter with.
+      isArray: false
+      name: quarantined_as_spam
+      predefined:
+      - 'True'
+      - 'False'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Quarantine status to filter with.
+      isArray: false
+      name: quarantine_status
+      predefined:
+      - POLICY
+      - AMP
+      - AV
+      - UNCLASSIFIED
+      - DLP
+      - OUTBREAK
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Url reputation to filter with
+      isArray: false
+      name: url_reputation
+      predefined:
+      - Malicious
+      - Suspicious
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Virus positive to filter with
+      isArray: false
+      name: virus_positive
+      predefined:
+      - 'True'
+      - 'False'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: |-
+        Domain name operator to filter with.
+        If you want to use domain filter, you need to use both - domain_name_operator, domain_name_value.
+      isArray: false
+      name: domain_name_operator
+      predefined:
+      - is
+      - begins_with
+      - contains
+      required: false
+      secret: false
+    - default: false
+      description: |-
+        Domain name value to filter with.
+        If you want to use domain filter, you need to use both - domain_name_operator, domain_name_value.
+      isArray: false
+      name: domain_name_value
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Is contained malicious urls filter
+      isArray: false
+      name: contained_malicious_urls
+      predefined:
+      - 'True'
+      - 'False'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Is contained natural urls filter
+      isArray: false
+      name: contained_neutral_urls
+      predefined:
+      - 'True'
+      - 'False'
+      required: false
+      secret: false
+    - default: false
+      description: |
+        Macro file types detected to filter with.
+        temlate of macro_file_types_detected -  Microsoft%20Office%20Files,Adobe%20Portable%20Document%20Format
+      isArray: false
+      name: macro_file_types_detected
+      required: false
+      secret: false
+    deprecated: false
+    description: Cisco Email Security search messages.
+    execution: false
+    name: cisco-email-security-messages-search
+    outputs:
+    - contextPath: CiscoEmailSecurity.Messages.attributes.direction
```

Comment:
'Messages' is still in plural, and also some other context outputs have the same problem.
Let's fix this please

Reply:
done